sys/linux/usbip.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <drivers/usb/usbip/usbip_common.h>
+
+resource usbip_server_fd[fd]
+
+syz_usbip_server_init(usb3 bool32, speed flags[usbip_device_speed]) usbip_server_fd
+write$usbip(fd usbip_server_fd, buffer ptr[in, usbip_header], len bytesize[buffer])
+
+type usbip_header_basic[PACKET_TYPE] {
+	command		const[PACKET_TYPE, int32be]
+	seqnum		int32be
+	devid		int32be
+	direction	flags[direction_flags, int32be]
+	ep		int32be
+} [packed]
+
+usbip_header_ret_submit {
+	status			int32be
+	actual_length		int32be
+	start_frame		int32be
+	number_of_packets	int32be[0:USBIP_MAX_ISO_PACKETS]
+	error_count		int32be
+} [packed]
+
+usbip_header_ret_unlink {
+	status	int32be
+} [packed]
+
+usbip_packet_ret_submit {
+	base_submit	usbip_header_basic[USBIP_RET_SUBMIT]
+	ret_submit	usbip_header_ret_submit
+} [packed]
+
+usbip_packet_ret_unlink {
+	base_unlink	usbip_header_basic[USBIP_RET_UNLINK]
+	ret_unlink	usbip_header_ret_unlink
+} [packed]
+
+usbip_header [
```

Comment:
Rename `usbip_header` to `usbip_header_something` so this doesn't confuse Dmitry's scripts.